### ISSUE TITLE
Bit.ly option using soon to be deprecated API key issues

### ISSUE THREAD

**Issue by snowbound** [fwenzel/copy-shorturl#77]
I have used the older style API key and user account in various shortening apps over the years and it worked. I copied the API_Key that I have always used  from my bit.ly account and pasted it into the extension and saved  the settings.

I then spent 20 minutes wondering why I kept seeing 
![capture](https://user-images.githubusercontent.com/596147/31560655-c762f61c-b01a-11e7-9d0b-c9aafad9f00d.JPG)

I gave up using the API_Key and had to to go through the OAuth token generation process and embedding the token into the https://api-ssl.bitly.com/v3/shorten?format=txt&access_token=<accesstoken>&longUrl=%URL%

As mentioned in the documentation http://copy-shorturl.readthedocs.io/en/latest/serviceurl.html

**Comment by fwenzel** [comment]
Good call that the old documentation might be out of date. I didn't update it when I turned the add-on into a webextension.

I'll also make sure to bubble errors into the console so at least you'll know what happened.

Side note: I am using bitly with an API key right now with no problems.

**Comment by fwenzel** [comment]
Oh also, OAuth Token (the thing I am linking to in the copy shorturl prefs) is obviously what you need

**Comment by fwenzel** [comment]
To confirm, did you generate a "generic access token" with bitly?

**Comment by snowbound** [comment]
Yes I originally started to create the Registered OAuth Application via the Bit.ly menu but realized that it seemed overly complicated for what I needed. That was when I discovered via bit.ly that a "generic access token" was more suitable. Perhaps the online documentation could indicated that?

**Comment by fwenzel** [comment]
Yeah has little to do with more suitable -- the implementation I wrote is for what bitly calls "generic access token".

**Comment by fwenzel** [comment]
I'll keep this Issue open to adjust the docs!
